extensions/1password/src/v7/utils.ts

```diff
@@ -9,6 +9,7 @@ import { readFileSync } from "fs";
 const cache = new Cache();
 
 const ITEMS_CACHE_NAME = "@items";
+export const SORTING_METHOD = getPreferenceValues<Preferences>().sortingMethod || "title";
```

Comment:
Where do you get `Preferences` from?

Reply:
I don't think this is solved yet? 

![Raycast-gpjHxwza@2x](https://user-images.githubusercontent.com/38033107/222112948-91d8d64a-27ba-4988-90f2-91c29ae89bf9.png)

This defines the type, but there is no type?